chacra/tests/async/test_recurring.py

```diff
@@ -37,16 +37,40 @@ def test_gets_rid_of_old_repos(self, session, fake, monkeypatch):
         recurring.purge_repos(_now=self.now)
         assert Repo.query.all() == []
 
-    def test_does_not_get_rid_of_old_repos_configured_in_days(self, session, fake, monkeypatch):
-        conf.purge_rotation = {'ceph': {'firefly': {'days': 70}}, '__force_dict__': True}
+    def test_does_not_get_rid_of_old_repos_by_ref_configured_in_days(self, session, fake, monkeypatch):
+        conf.purge_rotation = {'ceph': {'ref': {'firefly': {'days': 70}}}, '__force_dict__': True}
         fake_datetime = fake(utcnow=lambda: self.old, now=self.now)
         monkeypatch.setattr(datetime, 'datetime', fake_datetime)
         session.commit()
         recurring.purge_repos(_now=self.now)
         assert len(Repo.query.all()) == 1
 
-    def test_does_not_get_rid_of_old_repos_configured_with_offset(self, session, fake, monkeypatch):
-        conf.purge_rotation = {'ceph': {'firefly': {'keep_minimum': 1}}, '__force_dict__': True}
+    def test_does_not_get_rid_of_old_repos_by_ref_configured_with_offset(self, session, fake, monkeypatch):
```

Comment:
these tests are great. Can you add the reverse use case, when repos are removed? I only see tests for repos that don't get removed.